src/game/models/player.rs

```diff
@@ -39,7 +39,10 @@ const THRUST_INCREMENT: f64 = 5.0;
 impl Player {
     pub fn new(window_size: Size) -> Player {
         Player {
-            pos: Vector { x: 10.0, y: 10.0 },
+            pos: Vector {
+                x: window_size.width as f64 / 2.0,
```

Comment:
We aren't; we're setting the player's starting position to the center of the screen.